ui/src/app/views/app/component.ts

```diff
@@ -190,4 +192,9 @@ export class AppComponent implements OnInit {
     });
   }
 
+  private async loadConfig() {
+    const config = await configApi().getConfig();
```

Comment:
Please get this via `ServerConfigService.getConfig()` to avoid the extra request:

https://github.com/all-of-us/workbench/blob/00a906d0e936cead8ca2ea7cf5b63cdf324e86cc/ui/src/app/services/server-config.service.ts#L14

You might need to change the type or coerce the type for now (while we're still mixed React/Angular)